docs/api/animation/AnimationAction.html

```diff
@@ -11,121 +11,363 @@
     <h1>[name]</h1>
 
     <div class="desc">
-    An AnimationAction schedules clip playback on specific objects.
+        AnimationActions schedule the performance of the animations which are stored in
+        [page:AnimationClip AnimationClips].<br /><br />
+        
+        Note: Most of AnimationAction's methods can be chained.<br /><br />
+        
+        For an overview over the different elements of the three.js animation system see the
+        "Animation System" article in the "Next Steps" section of the manual.
     </div>
-
+    
 
     <h2>Constructor</h2>
-
+    
 
     <h3>[name]( [page:AnimationMixer mixer], [page:AnimationClip clip], [page:Object3D localRoot] )</h3>
-
-
+    <div>       
+        [page:AnimationMixer mixer] - the *AnimationMixer* that is controlled by this action.<br />
```

Comment:
Oh, ok I see your explanation later on. I think my mental model is just a bit different here, but it's fine as you've written it.